src/libAtomVM/opcodesswitch.h

```diff
@@ -6102,6 +6102,18 @@ HOT_FUNC int scheduler_entry_point(GlobalContext *glb)
                 break;
             }
 
+            case OP_RAW_RAISE: {
+
+                TRACE("raw_raise/0\n");
+
+                #ifdef IMPL_EXECUTE_LOOP
+                    // we assume x[0]-x[2] contain valid error information
```

Comment:
LGTM. This comment could be more assertive. This is an optimizatiob from the compiler where we don't need to call `stacktrace_create_raw` here because the stack trace has already been created and set in x[2].